src/PresentationalComponents/PlainTextWithLink.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TextVariants } from '@patternfly/react-core';
+
+export const PlainTextWithLink = ({ label, linkText, linkUrl }) => {
+    return (
+        <Text component={ TextVariants.p }>
+            {label}
+            <Text className="pf-u-ml-md" component={ TextVariants.a } target="_blank" href={ linkUrl }>{linkText}</Text>
```

Comment:
I would add `rel="noreferrer noopener"` for better security.

Reply:
Oh thanks! I forgot about that!